data/company.yaml

```
@@ -0,0 +1,62 @@
+    
+    - name: "Top Ruby on Rails Development Company"
+      issuer: "TopDevelopers.co"
+      year: "2022"
+      link: "https://www.topdevelopers.co/"
+
+  press_mentions:
+    - title: "JetThoughts Named Top Development Partner"
+      publication: "Tech Industry Review"
+      link: "#"
+      date: "2023-09-15"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)